### PR TITLE
EDU-932 Update to nodejs 18

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-932

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,73 +1,16 @@
 {
-  "extends": ["eslint:all", "plugin:import/recommended"],
+  "extends": [
+    "eslint:recommended"
```
Comment (ahelmberger) [inline .eslintrc]: You removed the `import` plugin from the config but it's still installed as a `devDependency`. Could we keep using it, I think it is pretty helpful?

Comment (adrianaluchian) [reply .eslintrc]: Will add it back, funny that I had copied over this whole setup from another repo we have, need to check which.

Comment (ahelmberger) [reply .eslintrc]: I know we are not consistent on this :-(

Comment (adrianaluchian) [reply .eslintrc]: Updated

Comment (ahelmberger) [reply .eslintrc]: I think it's still missing in the `extends` array :thinking:

Comment (adrianaluchian) [reply .eslintrc]: Of course it is 🤦‍♀️ Added, thanks

Comment (ahelmberger) [reply .eslintrc]: :+1:

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**